src/cascadia/TerminalApp/AboutDialog.h

```diff
@@ -1,36 +1,36 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like some of the files weren't utf-8 before, and I think they got re-saved as utf8. 

Go here to disable the whitespace deltas:
https://github.com/microsoft/terminal/pull/16837/files?w=1

Reply:
Line endings changed - pushed a fix.